packages/vuetify/src/components/VAppBar/VAppBar.tsx

```diff
@@ -0,0 +1,159 @@
+// Styles
+import './VAppBar.sass'
+
+// Components
+import { VImg } from '@/components'
```

Comment:
should we do `'@/components/VBtn'` like with composables?

Reply:
We only decided against an index for composables because of how many things they can/do export. The intent is destructure elsewhere.